src/applications/mhv-secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -264,14 +273,34 @@ const ComposeForm = props => {
 
   useEffect(
     () => {
-      if (
-        messageInvalid ||
-        (isSignatureRequired && (signatureInvalid || checkboxInvalid))
-      ) {
-        focusOnErrorField();
-      }
+      const handleEffect = async () => {
+        const selectedTG = recipientsList.find(
+          recipient => recipient.triageTeamId === +selectedRecipientId,
+        );
+        if (
+          alertDisplayed &&
+          !alertHasFocused &&
+          alertRef.current &&
+          selectedTG
+        ) {
+          await focusElement(selectedTG?.name);
+          focusElement(alertRef.current);
+          setAlertHasFocused(true);
+        } else if (messageInvalid) {
+          focusOnErrorField();
+        }
+      };
+      handleEffect();
```

Comment:
did you run in a particular scenario where you would not benefit from calling the code in `handleEffect` const directly without wrapping in a const?

Reply:
Actually, yes! I would get a strange `DOMException` error that failed to execute the `'querySelector'` within the `focusElement()` helper.  I chose to refactor this code this way so we can use `async await` anyway so that the SR focus would announce the selected `recipient` first before announcing the `signature-alert`.